<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReagentShelfListRowUi</class>
 <widget class="QWidget" name="ReagentShelfListRowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,10,47">
   <item>
    <widget class="QLabel" name="lblPos">
     <property name="text">
      <string>Pos(1-18)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxNonEmpty">
     <property name="text">
      <string>使用</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboReagent"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
